langchain/vectorstores/chroma.py

```diff
@@ -363,6 +363,30 @@ def update_document(self, document_id: str, document: Document) -> None:
             metadatas=[metadata],
         )
 
+    def update_document_batch(
```

Comment:
while we're here should we simplify `update_document` to use this? eg
```python
def update_document(...):
  return self.update_document_batch([docuent_id], [document])
```

Reply:
Brilliant!

Reply:
I can modify this as well on this pull request if you want @dev2049